VERSIONING.md

```diff
@@ -12,76 +12,92 @@ Minor releases contain new smaller features. Minor releases are fully backward-c
+
+feature-{IssueShortTitle}
+
+where {ShortTitle} is written using [UpperCamelCase](https://es.wikipedia.org/wiki/CamelCase)
+
+Examples
+- feature-DisableDragAndDrop
+- feature-RoleDirective
 
-First, if you wish to continue to release maintenance fixes for the feature release made before the recent one, then you must create another branch to track commits for that previous release.
+### Release branches
 
-To do this, the feature branch is copied to another branch named with the previous release version number.
+After a release, it is needed to manage a release branch.
 
-Maintenance branches will be named as Major.Minor.x, for example 5.12.x
+Release branches will be named as release-Major.Minor.x, for example release-5.12.x
 
-Rules to 'Require pull request reviews before merging' must be applied to every maintenance branch. You can use wildcards in order to do so: \[1-9]\*.* will do the work
+Rules to 'Require pull request reviews before merging' must be applied to every release branch. You can use wildcards in order to do so: release-* will do the work
 
-### Branching procedure
+#### Branching procedure
 
 For a new change:
 
-- If it is minor change, the contributor should create a new maintenance branch, for example 5.13.x
-- If it is major change (this is usually something that you should do from the master branch), the contributor should create a new maintenance branch, for example 6.0.x.
+- If it is minor change, the contributor should create a new release branch, for example 5.13.x
+- If it is major change (this is usually something that you should do from the master branch), the contributor should create a new release branch, for example 6.0.x.
 - You do not need to create a new branch for a patch change.
 
-In order to add changes to a maintenance branch:
+In order to add changes to a release branch:
 
- - Create a new working branch from the maintenance branch.
- - Commit the changes to the new working branch.
- - Create a Pull Request from the working branch to the maintenance branch.
+ - Create a new feature branch from the release branch.
+ - Commit the changes to the new feature branch.
+ - Create a Pull Request from the feature branch to the release branch.
  - Once the Pull Request is approved it is time to merge the changes. Will be the submitter the responsible for merging.
- - Finally, the working branch could be removed.
+ - Finally, the feature branch could be removed.
 
 > In Angular, before creating the PR, be sure to update, as needed, the npm library version in the package.json file.
 
-### Bug fixing policy
+## Bug fixing policy
 
 When addressing a bug detected on a branch, it should be fixed at least on:
-- The maintenance branch where needed
+- The release branch where needed
 - The master branch
 
 Detected bugs should be tracked as repository issues with the 'Bug' tag. So, their history should indicate for which versions the defect has been fixed.
 
```

Comment:
After the conversation we had, I think that this policy needs to be detailed to cover the different scenarios/needs. I would suggest that defects should be first classified according its severity (i.e, low, medium and high) and then a different policy should apply depending on that. For instance:
* High severity: fixed on all the active release branches and on master
* Medium severity: fixed on release branch where found and all further active release branches
* Low severity: fixed only on the release branch where found and on master